packages/angular-test-app/src/preview-examples/echarts-bar-horizontal-stacked.ts

```diff
@@ -16,6 +16,7 @@ import { BarSeriesOption, EChartsOption } from 'echarts';
 @Component({
   selector: 'app-example',
   templateUrl: './echarts-bar-horizontal-stacked.html',
+  styles: [`@import 'example-styles/dist/charts.css'`],
```

Comment:
Dont to inline style to import a css use styleUrl/styleUrls (https://v17.angular.io/api/core/Component#styleUrls)

Counts for every new angular component you created